flutter/pubspec.yaml

```diff
@@ -10,9 +10,13 @@ environment:
 
 dependencies:
```

Comment:
I think the formatter will complain about this later, so might as well fix it now. Please order the packages alphabetically.